lang/cs/Org.Apache.Reef.DotNet.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26228.10
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.Reef.Utilities.DotNet", "Org.Apache.REEF.Utilities\Org.Apache.Reef.Utilities.DotNet.csproj", "{FE4CEA75-2F26-4C97-B256-375AA698B4A9}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Org.Apache.REEF.Tang.DotNet", "Org.Apache.REEF.Tang\Org.Apache.REEF.Tang.DotNet.csproj", "{416CC00B-790A-49BE-B5A5-015B0AF17A92}"
+	ProjectSection(ProjectDependencies) = postProject
+		{FE4CEA75-2F26-4C97-B256-375AA698B4A9} = {FE4CEA75-2F26-4C97-B256-375AA698B4A9}
+	EndProjectSection
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{FE4CEA75-2F26-4C97-B256-375AA698B4A9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
I actually dont know what those should be. Technically these are libraries and can target any platform that gets built. So they can be Any CPU, but seeing how we will never build 32 bit binaries, it makes sense to just target x64. Plus I think my programmer soul would scream silently if I saw Any CPU with other binaries that were targeting x64.  So I went ahead and updated the 
solution  to have only x64 projects.